pkgs/tools/misc/iotools/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub
+, gnumake, gcc10
+}:
+
+stdenv.mkDerivation rec {
+  pname = "iotools";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "adurbin";
+    repo = pname;
+    rev = "18949fdc4dedb1da3f51ee83a582b112fb9f2c71";
```

Comment:
```suggestion
    rev = "v{version}";
```

Reply:
If it doesn't hurt too much, I would like to use the git revision since iotools got many improvements since v1.6.

Reply:
Then the version should be changed to unstable-XXXX-XX-XX where XXXX-XX-XX is the date of the commit.

Reply:
Can I also use the first chars of the git revision id?

Reply:
I am not sure. This is usually only done if upstream does it.

Reply:
> Can I also use the first chars of the git revision id?

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".